src/Workspaces/VisualBasic/Portable/Formatting/Engine/Trivia/TriviaDataFactory.ComplexTrivia.vb

```diff
@@ -81,6 +81,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
 
                 Debug.Assert(Me.SecondTokenIsFirstTokenOnLine OrElse beginningOfNewLine)
 
+                If Me.OptionSet.GetOption(FormattingOptions.UseTabs, LanguageNames.VisualBasic) Then
```

Comment:
This fix solves the issue but I think the correct location to provide the fix would be, inside of `CodeShapeAnalyzer.OnWhiteSpace(trivia, index)` which is eventually called by `CodeShapeAnalyzer.ShouldFormatMultiLine`.  With the current fix, we will end up formatting the Complex trivia every time we see the `UseTabs` is enabled irrespective of the content of the trivia. ComplexTrivia formatting is an expensive operation and it should not be done all the time.